src/run.js

```diff
@@ -46,13 +48,20 @@ async function run({
     return new Promise((res, rej) => {
       const nwProcess = child_process.spawn(
         path.resolve(nwDir, util.EXE_NAME[platform]),
-        [srcDir]
+        [srcDir, ...args],
+        { detached: true, stdio: "ignore" },
       );
 
+      nwProcess.unref();
```

Comment:
This statement prevents blocking the caller (usually `npm run nw ...`) which in turn exit immediately with code `1`.
Suggestion to remove and keep blocking the caller as it was done in 0.85.0 and below.